mods/cnc/rules/world.yaml

```diff
@@ -13,6 +13,126 @@
 		FogVariants: typea, typeb, typec, typed
 		OverrideFullShroud: full
 		OverrideFullFog: full
+	Locomotor@default:
```

Comment:
`@DEFAULT` for consistency please.

Reply:
Maybe we should actually remove these, to prevent modders (and us) from accidentally forgetting to give all mobile actors proper locomotors (and then not even noticing because the actors silently use those fallbacks).

Reply:
Removing them actually turned out to be a good idea, caught a typo that way. Also added YamlExceptions at rules loading time, because mismatches crash the utility before the lint rule can tell you the problem (which makes me wonder whether those exceptions make the lint rule obsolete).